src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+body {
+  margin: 0;
+  width: 100px;
+  height: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
``html`` and ``body`` tag, set styles for the entire page, if you set some styles for one, there is no need to duplicate them in the other.